WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -161,23 +167,51 @@ - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInte
     return nil;
 }
 
+- (NSString *)tableView:(UITableView *)tableView titleForFooterInSection:(NSInteger)section
+{
+    return nil;
+}
+
+- (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger)section
+{
+    return [UIView new];
+}
+
+- (UIView *)tableView:(UITableView *)tableView viewForFooterInSection:(NSInteger)section
+{
+    return [UIView new];
+}
+
+- (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSInteger)section
+{
+    return CGFLOAT_MIN;
+}
+
+- (CGFloat)tableView:(UITableView *)tableView heightForFooterInSection:(NSInteger)section
+{
+    return CGFLOAT_MIN;
+}
+
 - (CGFloat)tableView:(UITableView *)tableView estimatedHeightForRowAtIndexPath:(NSIndexPath *)indexPath
 {
     // Note:
     // Without an estimated height, UITableView will have an erratic behavior
-    return CommentsDefaultCellHeight;
+    return WPTableViewDefaultRowHeight;
 }
 
 - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
 {
-    Comment *comment = [self.tableViewHandler.resultsController objectAtIndexPath:indexPath];
-    return [CommentsTableViewCell rowHeightForContentProvider:comment andWidth:WPTableViewFixedWidth];
+    CommentsTableViewCell *tableViewCell = [tableView dequeueReusableCellWithIdentifier:CommentsLayoutIdentifier];
+    [self configureCell:tableViewCell atIndexPath:indexPath];
```

Comment:
It might be an optimization to retain and reuse a single instance vs continually dequeuing new cells. You might get the same cell back but you'd still have the dequeuing as overhead.   Just a thought.